htdocs/admin/ticket.php

```diff
@@ -358,7 +361,7 @@
 
 
 
-/*
+/********************************
```

Comment:
/*** must be avoid.
Use only 
/*   for any comment
or
/**     for comment on method / class